src/libs/functions/urlHelpers.js

```diff
@@ -40,7 +40,41 @@ export const checkIsRedirectOutside = ({ currentUrl, destinationUrl }) => {
 
   return false
 }
+/**
+ *
+ * @param {object} params
+ * @param {string} params.cozyUrl The URL of the cozy, from client.stackclient.uri
+ * @param {string} params.destinationUrl The URL we want to redirect the user
+ * @param {'flat' | 'nested'} params.subdomainType - the Cozy's subdomain type
+ * @returns {boolean} True is the destination is not on the cozy
+ */
+export const checkIsCozyUrlIsTheDestination = ({
```

Comment:
The function's name seems off here.

First it has 2 `is` which is hard to understand. Shouldn't the first `is` be an `if`?

But even with that, if I call `checkIfCozyUrlIsTheDestination` I expect to get `true` if the sourceURL and the destinationURL are the same. But here `true` means that there are different (so we are leaving the cozy). So I would reverse the name or the logic.

Proposal: rename it to something like `checkIsRedirectOutsideCurrentCozy`

Reply:
> First it has 2 is which is hard to understand. Shouldn't the first is be an if?

Yep. Sorry for that. 

You right, I'll change the logic. But in fact there is already a named `checkIsRedirectOutside` that doesn't not really what I wanted. I'll think of something.

Reply:
What if you inverse the logic and name it `isSameCozy`?